src/H5Tinit_float.c

```diff
@@ -0,0 +1,552 @@
+                if (_v1 != _v2)                                                                              \
+                    _pad_mask[_i] |= _byte_mask;                                                             \
+                H5_GCC_CLANG_DIAG_ON("float-equal")                                                          \
+                _buf1[_i] ^= _byte_mask;                                                                     \
+            }                                                                                                \
+                                                                                                             \
+        /* Byte Order */                                                                                     \
+        for (_i = 0, _v1 = (TYPE)0.0L, _v2 = (TYPE)1.0L; _i < (int)sizeof(TYPE); _i++) {                     \
+            _v3 = _v1;                                                                                       \
+            _v1 += _v2;                                                                                      \
+            _v2 /= (TYPE)256.0L;                                                                             \
+            memcpy(_buf1, (const void *)&_v1, sizeof(TYPE));                                                 \
+            memcpy(_buf3, (const void *)&_v3, sizeof(TYPE));                                                 \
+            _j = H5T__byte_cmp(sizeof(TYPE), _buf3, _buf1, _pad_mask);                                       \
+            if (_j >= 0) {                                                                                   \
+                INFO.perm[_i] = _j;                                                                          \
+                _last         = _i;                                                                          \
+            }                                                                                                \
+        }                                                                                                    \
+        H5T__fix_order(sizeof(TYPE), _last, INFO.perm, &INFO.order);                                         \
+                                                                                                             \
+        /* Implicit mantissa bit */                                                                          \
+        _v1       = (TYPE)0.5L;                                                                              \
+        _v2       = (TYPE)1.0L;                                                                              \
+        INFO.imp  = H5T__imp_bit(sizeof(TYPE), INFO.perm, &_v1, &_v2, _pad_mask);                            \
+        INFO.norm = INFO.imp ? H5T_NORM_IMPLIED : H5T_NORM_NONE;                                             \
```

Comment:
I don't see this line ("INFO.norm = ...") in the H5detect.c source - where did it come from?

Reply:
I added it because it was easier to set the norm here than do the string munging in H5detect.

Reply:
Found it, resolving this comment.